fil_finder/pixel_ident.py

```diff
@@ -64,8 +64,10 @@ def isolateregions(binary_array, size_threshold=0, pad_size=5,
     for n in range(1, num + 1):
         x, y = np.where(labels == n)
         # Make an array shaped to the skeletons size and padded on each edge
-        shapes = (x.max() - x.min() + 2 * pad_size,
-                  y.max() - y.min() + 2 * pad_size)
+        # the +1 is because, e.g., range(0, 5) only has 5 elements, but the
+        # indices we're using are range(0, 6)
+        shapes = (x.max() - x.min() + 1 + 2 * pad_size,
+                  y.max() - y.min() + 1 + 2 * pad_size)
```

Comment:
No, not needed in that case, but I think padding should be optional.  Either that, or checks for valid pad #s are needed earlier.